internal/schema/dynamic/dynamic.go

```diff
@@ -23,22 +32,44 @@ type StringReader struct {
 }
 
 func (r FileReader) Read() (map[string]schema.Schema, error) {
+	dynamicResourcesSchema := "https://raw.githubusercontent.com/hashicorp/terraform-provider-tfcoremock/main/schema/dynamic_resources.json"
```

Comment:
Could we use [embed](https://pkg.go.dev/embed) (or something similar) to bake this schema into the binary? Not a huge deal but it does feel odd for a provider to reach out to GitHub at runtime.